tests/acceptance/features/apiSpaces/editPublicLinkOfSpace.feature

```diff
@@ -16,16 +16,18 @@ Feature: A manager of the space can edit public link
     And using spaces DAV path
     And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
     And user "Alice" has created a space "edit space" with the default quota using the Graph API
-    And user "Alice" has created a public link share of the space "edit space" with settings:
-      | permissions | 1                        |
-      | password    | %public%                 |
-      | expireDate  | 2040-01-01T23:59:59+0100 |
-      | name        | someName                 |
+    And user "Alice" has created the following space link share:
+      | space              | edit space               |
+      | permissionsRole    | view                     |
+      | password           | %public%                 |
+      | expirationDateTime | 2040-01-01T23:59:59.000Z |
+      | displayName        | someName                 |
     And user "Alice" has uploaded a file inside space "edit space" with content "some content" to "test.txt"
 
 
   Scenario Outline: manager of the space can edit public link.
     Given using OCS API version "2"
+    And using SharingNG
     When user "Alice" updates the last public link share using the sharing API with
```

Comment:
this can go to the background
```suggestion
      | displayName        | someName                 |
    And user "Alice" has uploaded a file inside space "edit space" with content "some content" to "test.txt"
    And using SharingNG


  Scenario Outline: manager of the space can edit public link.
    Given using OCS API version "2"
    When user "Alice" updates the last public link share using the sharing API with
```